frontend-engineer/catalog-product-list.md

```diff
@@ -28,14 +28,24 @@ id is the product SKU, image contains a full URL, the price is in cents and the
 
 ## Technical constraints:
 
-* You can use any programming language, tools and frameworks you see fit, preferably React, Redux, Styled components💅. 
-* You need to show some knowledge of HTML5, modern JavaScript(ES6).
-* You need to provide a straight-forward method of running your project.
+* You should use  React, Styled components💅, state management is a matter of your choice.
+* You need to show some knowledge of HTML5, modern JavaScript(ES6) or TypeScript.
+* You will provide a working Dockerfile to run your project
+* You will provide some methods to filter the list of products by brand and type.
```

Comment:
let's move the new point "methods to filter" to the paragraph above "Required functionality"